draft-davidben-tls-merkle-tree-certs.md

```diff
@@ -470,7 +470,7 @@ For each batch in the "issued" state, the CA maintains the following batch state
 
 * The tree head, a hash computed over this list, described in {{building-tree}}.
 
-* A window signature computed as described in {{signing}}.
+* A validity window signature computed as described in {{signing}}.
```

Comment:
I like window signature better. We're not going to sign other windows, are we? So perhaps just window signature?

Reply:
(Marking unresolved just because it sounds like we're still deciding the terminology.)

Yeah, "window signature" would work too. My thinking around "batch's signature" was that just that we currently define this object by saying:

> For each batch in the "issued" state, the CA maintains the following batch state:
>
> * The list of assertions certified in this batch.
>
> * The tree head, a hash computed over this list, described in {{building-tree}}.
>
> * A signature over the corresponding validity window, described in {{signing}}.

And since we're associating it with the batch, it seems plausible to say "the batch's signature".

To that end, we still have a couple instances of "validity window signature" which is now a term without a definition. Whichever we pick, I think we want to be consistent between where it's defined and used.

But if we say the batch has a "window signature" and then elsewhere we say "batch's window signature", that works too. "validity window signature" would also work, I just think it's a little wordy and hard to parse. But this is a very soft preference. I don't feel very strongly and would be happy with "validity window signature" if that's the cleanest.

Reply:
I think the draft has many moving parts, and `validity window signature` to my ear seems easier to understand even if it's wordy, but perhaps we should just see what people that read the draft for the first time prefer.

Reply:
(We could merge and change later again — that'll reduce merge conflicts.)

Reply:
> Sorry for the churn

These small details are important, and it's not perfect yet. Let's revisit it later, despite the churn :).